scala/scala.bzl

```diff
@@ -554,10 +572,13 @@ def scala_test_suite(
         name,
         srcs = [],
         visibility = None,
+        use_short_names = False,
```

Comment:
I just added one: `//test:LongTestSuiteNamesSuite`

Without `use_short_name = True` it will fails with:
```
Executing tests from //test:LongTestSuiteNamesSuite_test_suite_longnames_looooooongnaaaaaaame_anooooootherlooooooooongname_anooooootherlooooooooongname2_anooooootherlooooooooongname3_LongNamesTest.scala
-----------------------------------------------------------------------------
Error: Current working directory has a path longer than allowed for a
Win32 working directory.
Can't start native Windows application from here.
```